### ISSUE TITLE
Itch on Meta Quest Browser shows the "mobile" version with standard browser size

### ISSUE THREAD

**Issue by SignorPipo** [itchio/itch.io#1384]
As you can see in the attached video, the Meta Quest browser shows the page in the "mobile" version, that is with the cover image under the running game, and also does not entirely show the full running game (not really visible here since it's black, but for example the VR button should be centered, which is not due to the fact that part of the game is cut on the right).

The game could be entirely shown by reducing the screen size, but it basically becomes very small, while as u can see adjusting the browser just a little bit makes everything work as expected: the game is fully shown and the cover image is hidden, showing the description directly below the game.

https://user-images.githubusercontent.com/16962655/199736205-9e79673a-4a2f-4d1e-9d06-25a449f97e6f.mp4

**Comment by davehill00** [comment]
More info on Quest Browser default panel size is available here: https://developer.oculus.com/documentation/web/browser-specs/#browser-window-size

TL;DR - Quest Browser defaults to 1000px wide.
